src/Services/TestimonialsApiService.js

```diff
@@ -0,0 +1,27 @@
+import {getRequest, postRequest, putRequest, deleteRequest} from '../../Services/publicApiService';
+const endPoint = `${process.env.REACT_APP_API_TESTIMONIALS}`;
```

Comment:
Acá no hace falta interpolación de variables:

```suggestion
const endPoint = process.env.REACT_APP_API_TESTIMONIALS
```

Reply:
Ah, y endpoint es una sla palabra así que no hace falta el camel case. Reemplazarlo en la constante y donde se usa en las diferentes requests!